src/main/java/cs/f10/t1/nursetraverse/model/ModelManager.java

```diff
@@ -48,8 +48,8 @@
 
     //Appointment list
 
-    private final AppointmentBook baseAppointmentBook;
-    private final AppointmentBook stagedAppointmentBook;
+    private AppointmentBook baseAppointmentBook;
```

Comment:
I've not followed the changes on this class, why are these no longer final?

Reply:
Because it is assigned to in `changeBaseTo` method